tests/test_dfes.py

```diff
@@ -67,6 +276,181 @@ def test_wrap_long_lines(self):
         assert textwrap.dedent(expected) in ds
 
 
+class TestCreateDFE:
+    """
+    Tests for creating a DFE instance.
+    """
+
+    def test_default_dfe(self):
+        d = dfe.DFE(
+                id="test",
+                description="test",
+                long_description="test",
+            )
+        assert d.mutation_types == []
+        assert d.proportions == []
+        assert d.citations == []
+        assert d.qc_dfe == None
+
+    def test_create_dfe(self):
+        d = dfe.DFE(
+                id="test",
+                description="test",
+                long_description="test",
+                citations=["555"], 
+                mutation_types=[dfe.MutationType(), dfe.MutationType()], 
+                proportions = [0.5, 0.5]
+            )
+        assert d.id == "test"
+        assert d.description == "test"
+        assert d.long_description == "test"
+        assert len(d.citations) == 1
+        assert d.citations[0] == "555"
+        assert len(d.mutation_types) == 2
+        for mt in d.mutation_types:
+            assert isinstance(mt, dfe.MutationType)
+        assert len(d.proportions) == 2
+        for p in d.proportions:
+            assert p == 0.5
+        assert d.qc_dfe == None
+
+    def test_create_dfe_without_citations(self):
+        d = dfe.DFE(
+                id="test",
+                description="test",
+                long_description="test",
+                mutation_types=[dfe.MutationType(), dfe.MutationType()], 
+                proportions = [0.5, 0.5]
+            )
+        assert d.citations == []
+
+    def test_create_dfe_without_proportions(self):
+        d = dfe.DFE(
+                id="test",
+                description="test",
+                long_description="test",
+                mutation_types=[dfe.MutationType()], 
+            )
+        assert d.proportions == [1]
+
+    def test_create_bad_dfes(self):
+        with pytest.raises(TypeError) as e_info:
+            # id, description, long_description are required
+            d = dfe.DFE()
+        assert str(e_info.value) == "DFE.__init__() missing 3 required keyword-only arguments: 'id', 'description', and 'long_description'"
```

Comment:
```suggestion
        with pytest.raises(TypeError, match="required .* arguments"):
            # id, description, long_description are required
            d = dfe.DFE()
```

Reply:
Although we don't really need to test this behavior - this is part of the fundamental behavior of python.